apps/vc-api/docs/example-exchange-definitions/one-issuance-one-self-sign-exchange.md

```diff
@@ -0,0 +1,116 @@
+Sample Exchange Definition for a self-signed and issuance - Holder needs to prove control over DID, thus exchange definition of type DIDAuth is needed to get the Credential from Issuer and another exchange definition to present the obtained credential and self-signed credentials.
+
+`DIDAuth` exchange-definition :
+
+```json
+{
+    "exchangeId": "286bc1e0-f1bd-488a-a873-8d71be3c690e",
+    "query": [
+      {
+        "type": "DIDAuth",
+        "credentialQuery": []
+      }
+    ],
+    "interactServices": [
+      {
+        "type": "MediatedHttpPresentationService2021"
+      }
+    ],
+    "callback": [
+      {
+        "url": "https://webhook.site/efb19fb8-2579-4e1b-8614-d5a03edaaa7a"
+      }
+    ],
+    "isOneTime":true
+}
+```
+
+`PresentationDefinition` exchange Definition.
```

Comment:
@nichonien I think that we only need one exchange here. The DID auth is not needed because proof over the DID is provided in the VP.

```suggestion
Sample Exchange Definition for a credential issuance which requires presentation of a self-signed credential. The Holder's proof control over the DID in the credential is provided via the proof on the verifiable presentation.
```

Reply:
Thanks for the input @jrhender, I was thinking differently. My understanding of the requirement was to obtained one credential from authority and other is self-signed, these two are then wrapped in a VP and shared with the verifier.

Reply:
@nichonien No, I don't think so. I'm not sure what you mean by "above mentioned scenario" but the file that we are commenting on here is "issuance and self-signed credential" and for issuance the exchange needs to have a "Mediated" interaction type, not "Unmediated". In the link you provided, the interaction type is "Unmedidated", so it is not suitable for "issuance and self-signed credential", IMO.